kibitzr/storage.py

```diff
@@ -83,6 +83,17 @@ def ensure_repo_exists(self):
             self.git.config("user.email", "you@example.com")
             self.git.config("user.name", "Your Name")
 
+    @staticmethod
+    def clean():
+        """Clean git repo"""
+        storage_dir = PageHistory.STORAGE_DIR
+        if os.path.isdir(storage_dir):
+            for root, dirs, files in os.walk(storage_dir, topdown=False):
```

Comment:
What do you think about using [shutil.rmtree](https://docs.python.org/2/library/shutil.html#shutil.rmtree) here?

Reply:
Point given. Easier to read. Sent my modifications via an amend commit.